packages/api/server.js

```diff
@@ -27,9 +30,11 @@ app.post('/', (req, res) => {
     message: req.body.message ? req.body.message : 'hello world'
   });
 });
+var postBuildHandler = new PostBuildHandler(app, client);
```

Comment:
I would use `const` rather than `var` (if you're never going to reassign the value).

I like this approach of pulling the route binding into a class, I recommend coordinating with @cedpeters and using this approach for other routes.